### PR TITLE
HELM-19: Add option to use the grafana login, name or email instead of the datasource name

### PR DESCRIPTION
**Pull request by mvrueden**
* JIRA: https://issues.opennms.org/browse/HELM-19

### CODE REVIEW

File: src/datasources/fault-ds/partials/config.html [hunk 1]
```diff
@@ -1,2 +1,44 @@
 <datasource-http-settings current="ctrl.current">
 </datasource-http-settings>
+
+<h3 class="page-heading">OpenNMS FM Datasource Details</h3>
+<div class="gf-form-group">
+    <div class="gf-form-inline">
+        <gf-form-switch class="gf-form"
+                        label="Use grafana user"
+                        tooltip="Defines if on supported alarm operations the currently logged in
```
Comment (j-white) [inline src/datasources/fault-ds/partials/config.html]: I would rephrase this as:
Used to control whether operations on alarms are performed the data source user, or the user that is currently logged in to Grafana.

File: src/datasources/fault-ds/partials/config.html [hunk 2]
```diff
@@ -1,2 +1,44 @@
...
+                                 Grafana user or the datasource user is used to talk to the ReST endpoint.
+                                 Supported operations are escalating, clearing, un-/acknowleding alarms as
+                                 well as creating/updating a journal/memo.
+                                 NOTE: Only OpenNMS admin users are allowed to perform actions as another user."
```
Comment (j-white) [inline src/datasources/fault-ds/partials/config.html]: I would rephrase this as:
NOTE: The data source must be configured using the 'admin' user in order to actions as other users.

File: src/datasources/fault-ds/partials/config.html [hunk 3]
```diff
@@ -1,2 +1,44 @@
...
+                        label-class="width-11"
+                        switch-class="max-width-6">
+        </gf-form-switch>
+
+        <div class="gf-form" ng-show="ctrl.current.jsonData.useGrafanaUser">
+            <label for="{{ctrl.id}}-grafana-user-field" class="gf-form-label pointer">
+                Field
+                <info-popover mode="right-normal" position="top center">
+                    Defines which field of the Grafana User should be used.
+                    This allows to fine grane what shows up as the "Action-Performing User" in OpenNMS. <br/>
```
Comment (j-white) [inline src/datasources/fault-ds/partials/config.html]: I would rephrase this as:
This allows for fine grained control over the value displayed as the "Action-Performing User" in OpenNMS.

**Comment by mvrueden** [comment]
Thanks for the sprinkles. I updated them accordingly.
